### PR TITLE
Fix wrong /security redirection

### LINKED ISSUE #30676: /docs/reporting-security-issues/ and /security redirects to a [ko] page
**This is a Bug Report**

**Problem:**
https://kubernetes.io/docs/reporting-security-issues
and
https://kubernetes.io/security

redirect to a Korean localization page

I think content was moved and _redirects wasn't been updated

**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/docs/reporting-security-issues
and
https://kubernetes.io/security

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Resolves #30676

### CODE REVIEW

**Comment by rikatz** [comment]
Can we also change the redirects to:

```
/security/ /docs/reference/issues-security/security/ 301
```
to always redirect to English translation?

**Review by rikatz — CHANGES_REQUESTED** [review]
Can we also change the redirects to:
``` 
/security/ /docs/reference/issues-security/security/ 301
```
to always redirect to English translation?

**Comment by jihoon-seo** [comment]
@rikatz I reflected your comment. 😊

**Review by rikatz — APPROVED** [review]
/lgtm
/cc @sftim @jimangel @divya-mohan0209

File: static/_redirects [hunk 1]
```diff
@@ -227,7 +227,8 @@
 
 /docs/reference/kubernetes-api/labels-annotations-taints/ /docs/reference/labels-annotations-taints/ 301
 
-/docs/reporting-security-issues/     /security/ 301
+/docs/reporting-security-issues/     /docs/reference/issues-security/security/ 301
+/security/ /docs/reference/issues-security/security/ 301
```
Comment (sftim) [inline static/_redirects:231]: nit: I'd use a 302 here.
```suggestion
/security/ /docs/reference/issues-security/security/ 302
```

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**